charts/knative-serving-istio/Chart.yaml

```diff
@@ -1,35 +1,35 @@
 apiVersion: v2
-name: knative-serving-istio
+appVersion: v1.3.0
+dependencies:
+- alias: knativeServingCore
+  condition: knativeServingCore.enabled
+  name: knative-serving-core
+  repository: https://caraml-dev.github.io/helm-charts
+  version: 1.1.2
+- alias: istiod
+  condition: istiod.enabled
+  name: generic-dep-installer
+  repository: https://caraml-dev.github.io/helm-charts
+  version: 0.2.1
+- alias: base
+  condition: base.enabled
+  name: base
+  repository: https://istio-release.storage.googleapis.com/charts
+  version: 1.13.9
+- alias: istioIngressGateway
+  condition: istioIngressGateway.global.enabled
+  name: generic-dep-installer
+  repository: https://caraml-dev.github.io/helm-charts
+  version: 0.2.1
+- alias: clusterLocalGateway
+  condition: clusterLocalGateway.global.enabled
+  name: generic-dep-installer
+  repository: https://caraml-dev.github.io/helm-charts
+  version: 0.2.1
 description: Installs Knative-serving for Istio
-type: application
-version: 1.3.0
-appVersion: "v1.3.0"
 maintainers:
-  - name: caraml-dev
-    email: caraml-dev@caraml.dev
-dependencies:
-  - name: knative-serving-core
-    repository: "https://caraml-dev.github.io/helm-charts"
-    version: ~1.1.0
-    alias: knativeServingCore
-    condition: knativeServingCore.enabled
-  - name: generic-dep-installer
-    repository: "https://caraml-dev.github.io/helm-charts"
-    version: ~0.2.1
-    alias: istiod
-    condition: istiod.enabled
-  - name: base
-    repository: "https://istio-release.storage.googleapis.com/charts"
-    version: 1.13.4
-    alias: base
-    condition: base.enabled
-  - name: generic-dep-installer
-    repository: "https://caraml-dev.github.io/helm-charts"
-    version: ~0.2.1
-    alias: istioIngressGateway
-    condition: istioIngressGateway.global.enabled
-  - name: generic-dep-installer
-    repository: "https://caraml-dev.github.io/helm-charts"
-    version: ~0.2.1
-    alias: clusterLocalGateway
-    condition: clusterLocalGateway.global.enabled
+- email: caraml-dev@caraml.dev
+  name: caraml-dev
+name: knative-serving-istio
+type: application
+version: 1.3.8
```

Comment:
@pradithya : This automatic chart version bumper incremented this charts version to `1.3.8` after upgrading the chart dependencies. Would it still be okay? Asking this with ref to your previous [comment](https://github.com/caraml-dev/helm-charts/pull/69#issuecomment-1302959054).

Context about this chart bumper: https://github.com/caraml-dev/helm-charts/pull/58

Reply:
Why does it set 1.3.8 instead of 1.3.1?

Reply:
>Known caveat
>The script goes through all charts and checks if there is a newer version of the listed dependencies. For each new version of a dependent chart found, it creates an update-cli manifest and executes the command. For each execution, the chart gets a patch version upgrade. This can result in skipped patch versions if there are multiple dependency updates.

This is a known thing, Going forward with it for now, will look for a solution.

Reply:
I see. Got it!